### ISSUE TITLE
Selenium Chrome Driver error in jasmine-integration task

### ISSUE THREAD

**Issue by Marlena** [pivotal-cf/react-starter#2]
Although gulp foreman works like a dream, if I run "gulp" to initiate the default task, it runs through starting the jasmine-integration task where it dies with the stack trace pasted below.  This also happens if you run the jasmine-integration task on its own.

Doubled checked that Chrome is my default browser.

```
[20:56:57] Using gulpfile ~/Github/react-starter/gulpfile.js
[20:56:57] Starting 'spec-integration'...
[20:56:57] Starting 'clean-assets'...
[20:56:57] Finished 'clean-assets' after 12 ms
[20:56:57] Starting 'assets'...
[20:57:03] Version: webpack 1.12.11
         Asset     Size  Chunks             Chunk Names
application.js  2.28 MB       0  [emitted]  application
[20:57:03] Finished 'assets' after 5.5 s
[20:57:03] Starting 'assets-html'...
[20:57:03] Finished 'assets-html' after 4.4 ms
[20:57:03] Starting 'dev-server'...
[20:57:03] Finished 'dev-server' after 396 ms
[20:57:03] Starting 'wait-for-server'...
[20:57:03] Server started http://localhost:8000
[20:57:03] Finished 'wait-for-server' after 5.66 ms
[20:57:03] Starting 'jasmine-integration'...

Users/marlena/Github/react-starter/node_modules/selenium-standalone/lib/install.js:248
  fs.access(where, fs.R_OK | fs.X_OK, function(err) {
     ^
TypeError: Object [object Object] has no method 'access'
    at chmodChromeDr (/Users/marlena/Github/react-starter/node_modules/selenium-standalone/lib/install.js:248:6)
    at /Users/marlena/Github/react-starter/node_modules/selenium-standalone/node_modules/async/lib/async.js:699:13
    at iterate (/Users/marlena/Github/react-starter/node_modules/selenium-standalone/node_modules/async/lib/async.js:256:13)
    at /Users/marlena/Github/react-starter/node_modules/selenium-standalone/node_modules/async/lib/async.js:268:29
    at /Users/marlena/Github/react-starter/node_modules/selenium-standalone/node_modules/async/lib/async.js:40:16
    at /Users/marlena/Github/react-starter/node_modules/selenium-standalone/node_modules/async/lib/async.js:705:17
    at Object._onImmediate (/Users/marlena/Github/react-starter/node_modules/selenium-standalone/lib/install.js:223:5)
    at processImmediate [as _immediateCallback] (timers.js:330:15)
```

**Comment by charleshansen** [comment]
Sorry for the late response, I'm not 100% what the issue is, Selenium is always tricky. For what it is worth, it should be using firefox by default, not Chrome. Do you have firefox installed? Also, what OS are you using? I've only verified integration on OSX so far.

**Comment by Marlena** [comment]
Hey, no worries.

I git cloned at work today and figured out that it's a combination of jdk
version and having Firefox as the default.

When I initially logged the issue, I was on my home computer and suspect
that I need to update the jdk there.

I did, however, notice a new issue, which should probably go in a separate
pull request.  The latest version isn't pulling in foreman.  I had to npm
install that separately.  Perhaps it's an issue with the react tools?  I
can log that separately if it would be helpful.

Cheers,
Marlena
